### ISSUE TITLE
Test 230316.001.4ft

### ISSUE THREAD

**Issue by sailist** [sailist/image-classification#15]
|Experiment Name|Test Name|Start| End |
|---|---|---|---|
|track_semi.chmatch|230316.001.4ft|23-03-16-192051||

```bash
python train_semi.py --module=chmatch --config=config/semi/chmatch/stl10.yml --scan=best-2023.03.04 --seed=0 --device=0 --group=v4.03161911
```

## Parameters
```python
{'apply_da': True,
 'apply_h_fc': True,
 'apply_h_mask': True,
 'batch_count': 1024,
 'batch_size': 64,
 'c_classes': 2,
 'config': 'config/semi/chmatch/stl10.yml',
 'dataset': 'stl10',
 'device': 0,
 'ema': True,
 'epoch': 1024,
 'eval': {'batch_size': 100,
          'num_workers': 8,
          'pin_memory': True,
          'shuffle': True},
 'fc': True,
 'filter_max': 0.95,
 'filter_min': 0.05,
 'first_strong': True,
 'group': 'v4.03161911',
 'include_ssl': True,
 'k_inc_epoch': 100,
 'k_type': 'linear',
 'max_proportion': 0.8,
 'method': 'chmatch',
 'min_proportion': 0.05,
 'mixup_beta': 0.5,
 'mlen': 50000,
 'moco_bank': False,
 'moco_bank_size': 65535,
 'model': 'resnet18',
 'module': 'chmatch',
 'mu': 7,
 'n_classes': 10,
 'n_percls': 100,
 'optim': {'lr': 0.03,
           'momentum': 0.9,
           'name': 'SGD',
           'nesterov': True,
           'weight_decay': 0.0005},
 'pp_rate': 0.05,
 'pretrain': True,
 'pretrain_path': None,
 'proportion_thresh': True,
 'pseudo_thresh': 0.95,
 'qk_p': False,
 'record_matrix': True,
 'record_predict': True,
 'right_bar': 0.1,
 'scan': 'best-2023.03.04',
 'second_strong': False,
 'seed': 0,
 'sharpen': 0.5,
 'stl10_unlabeled': True,
 'test': {'batch_size': 100,
          'num_workers': 8,
          'pin_memory': True,
          'shuffle': False},
 'train': {'batch_size': 64,
           'num_workers': 8,
           'pin_memory': True,
           'shuffle': True},
 'w_alpha': 1,
 'w_beta': 1,
 'w_cs': 1,
 'w_gamma': 1,
 'with_coarse': False}
```

## Metrics

|Key|Value|
|--|--|
|Acc|0.90925|

Tensor metrics:
```
cm
None

```

```

```

## Full properties
<details>
<summary> Click Here </summary>
<code>
{'agent': nan,
 'backup': {'23-03-17-100938': {'backend': 'github',
                                'number': 16,
                                'repo': 'pytorch-lumo/image-classification-private'},
            '23-03-17-101638': {'backend': 'github',
                                'number': 17,
                                'repo': 'pytorch-lumo/image-classification-private'},
            '23-03-19-154956': {'backend': 'github',
                                'number': 13,
                                'repo': 'sailist/image-classification'},
            '23-03-19-155216': {'backend': 'github',
                                'number': 15,
                                'repo': 'sailist/image-classification'}},
 'command': nan,
 'commit_hex': nan,
 'dataset': 'stl10',
 'deprecated': nan,
 'eidx': nan,
 'end': NaT,
 'error': nan,
 'exception': nan,
 'execute': {'cwd': '~/python/image-classification-private',
             'exec_argv': ['train_semi.py',
                           '--module=chmatch',
                           '--config=config/semi/chmatch/stl10.yml',
                           '--scan=best-2023.03.04',
                           '--seed=0',
                           '--device=0',
                           '--group=v4.03161911'],
             'exec_bin': '/opt/conda/bin/python',
             'exec_file': 'train_semi.py',
             'repo': '~/python/image-classification-private'},
 'exp_name': 'track_semi.chmatch',
 'finished': nan,
 'git': {'commit': '5d99a07c',
         'dep_hash': 'dab991486c68d32f4e3768b42532488d',
         'repo': '~/python/image-classification-private'},
 'hooks': {'Diary': {'loaded': True, 'msg': ''},
           'FinalReport': {'loaded': True, 'msg': ''},
           'GitCommit': {'loaded': True, 'msg': ''},
           'LastCmd': {'loaded': True, 'msg': ''},
           'LockFile': {'loaded': True, 'msg': ''},
           'RecordAbort': {'loaded': True, 'msg': ''}},
 'lock': {'accelerate': '0.11.0',
          'decorator': '4.4.2',
          'fire': '0.4.0',
          'hydra': '1.3.2',
          'joblib': '1.1.0',
          'lumo': '0.15.0',
          'numpy': '1.21.6',
          'omegaconf': '2.2.2',
          'psutil': '5.9.4',
          'torch': '1.8.1+cu101',
          'torch.version.cuda': '10.1'},
 'logger_args': {'log_dir': .lumo/experiments/track_semi.chmatch/230316.001.4ft/l.0.2303161920.log'},
 'metric': nan,
 'metric_board': .lumo/blob/track_semi.chmatch/230316.001.4ft/board.sqlite/metric_board.sqlite',
 'n_percls': 100.0,
 'note': '',
 'params': {'apply_da': True,
            'apply_h_fc': True,
            'apply_h_mask': True,
            'batch_count': 1024,
            'batch_size': 64,
            'c_classes': 2,
            'config': 'config/semi/chmatch/stl10.yml',
            'dataset': 'stl10',
            'device': 0,
            'ema': True,
            'epoch': 1024,
            'eval': {'batch_size': 100,
                     'num_workers': 8,
                     'pin_memory': True,
                     'shuffle': True},
            'fc': True,
            'filter_max': 0.95,
            'filter_min': 0.05,
            'first_strong': True,
            'group': 'v4.03161911',
            'include_ssl': True,
            'k_inc_epoch': 100,
            'k_type': 'linear',
            'max_proportion': 0.8,
            'method': 'chmatch',
            'min_proportion': 0.05,
            'mixup_beta': 0.5,
            'mlen': 50000,
            'moco_bank': False,
            'moco_bank_size': 65535,
            'model': 'resnet18',
            'module': 'chmatch',
            'mu': 7,
            'n_classes': 10,
            'n_percls': 100,
            'optim': {'lr': 0.03,
                      'momentum': 0.9,
                      'name': 'SGD',
                      'nesterov': True,
                      'weight_decay': 0.0005},
            'pp_rate': 0.05,
            'pretrain': True,
            'pretrain_path': None,
            'proportion_thresh': True,
            'pseudo_thresh': 0.95,
            'qk_p': False,
            'record_matrix': True,
            'record_predict': True,
            'right_bar': 0.1,
            'scan': 'best-2023.03.04',
            'second_strong': False,
            'seed': 0,
            'sharpen': 0.5,
            'stl10_unlabeled': True,
            'test': {'batch_size': 100,
                     'num_workers': 8,
                     'pin_memory': True,
                     'shuffle': False},
            'train': {'batch_size': 64,
                      'num_workers': 8,
                      'pin_memory': True,
                      'shuffle': True},
            'w_alpha': 1,
            'w_beta': 1,
            'w_cs': 1,
            'w_gamma': 1,
            'with_coarse': False},
 'params.yaml': .lumo/experiments/track_semi.chmatch/230316.001.4ft/params.yaml',
 'params_hash': 'e555ef51eb48e6d6b9039f4928c170c3',
 'path': nan,
 'paths': {'blob_root': .lumo/blob',
           'cache_root': '/dataset/asr/lumo-log-yhz/.cache/lumo',
           'info_root': .lumo/experiments'},
 'pinfo': {'hash': '45121295d275a6a9d82305b7f3ef9a5e',
           'obj': {'argv': ['/opt/conda/bin/python',
                            'train_semi.py',
                            '--module=chmatch',
                            '--config=config/semi/chmatch/stl10.yml',
                            '--scan=best-2023.03.04',
                            '--seed=0',
                            '--device=0',
                            '--group=v4.03161911'],
                   'pid': 107,
                   'pname': 'python',
                   'pstart': 1678965646.66},
           'pid': 107},
 'progress': {'finished': False,
              'last_edit_time': '23-03-19-154510',
              'ratio': 0.5009765625,
              'start': '23-03-16-192051',
              'update_from': None},
 'rerun': nan,
 'seed': 0,
 'start': NaT,
 'table_row': .lumo/experiments/track_semi.chmatch/230316.001.4ft/metric.pkl',
 'tags': nan,
 'test_name': '230316.001.4ft',
 'trainer': nan,
 'trainer_frame': nan}
</code>
</details>
